components/02-elements/pager/_pager.scss

```diff
@@ -1,46 +1,61 @@
-$pager__padding                 : 15px !default;
-$pager__margin                  : 15px !default;
-$pager__item-size               : 48px !default;
-$pager__item-color--current     : $white !default;
-$pager__item-background--current: $color-primary !default;
-$pager__link-color              : $gray !default;
-$pager__link-color-hover        : $color-primary !default;
-$pager__icon-width              : 12px !default;
-$pager__icon-height             : 12px !default;
+$pager__list-padding                        : $spacer !default;
+$pager__item-margin                         : 0 $spacer;
+$pager__item--current-after-height          : $spacer;
```

Comment:
shouldn't be `$pager__item-height-after--current` ?

Reply:
I think that after should be first so I've changed it to : `$pager__item-after-height--current` what do you think ?

Reply:
actually, i'm not sure about our examples of variables 😁 If you think it's ok, keep your variable